src/content/capture/worker-process/js/worker.js

```diff
@@ -0,0 +1,18 @@
+/*
+*  Copyright (c) 2019 The WebRTC project authors. All Rights Reserved.
+*
+*  Use of this source code is governed by a BSD-style license
+*  that can be found in the LICENSE file in the root of the source
+*  tree.
+*/
+
+'use strict;'
+
+onmessage = function(e) {
+  console.log('got a message');
```

Comment:
there is still a console.log statement here

Reply:
Oops. Sorry.